buster/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# SOURCE: https://github.com/William-Yeh/docker-ansible/blob/master/debian9-onbuild/Dockerfile
+FROM python:3.8-slim-buster
```

Comment:
same here: I would prefer to have `debian9` as identifier, rather than the code name.

and what does `slim` mean? can we omit it?

btw. it seems that the source your referring also has `debian9` in the path.